elementary/monitor/data_monitoring/selector_filter.py

```diff
@@ -110,6 +110,7 @@ def _parse_selector(self, selector: Optional[str] = None) -> SelectorFilterSchem
                     )
                 else:
                     logger.error(f"Could not parse the given -s/--select: {selector}")
```

Comment:
Why not raise an exception?
If a user specified a selector and it's invalid, I think we should fail rather than run without filters.

Reply:
I agree but this was intentionally done this way and I didn't want to change this behavior